test/support/mockConfig.js

```diff
@@ -7,6 +7,7 @@ const operationOutcomeInvalid = require('../fixtures/OperationOutcome/OperationO
 const sheResponse = require('../fixtures/Bundle-DDCC-TX-SHE-response-example-1.json');
 const ddccDocument = require('../fixtures/Bundle-DDCC-Document-example.json');
 const rhcResponse = require('../fixtures/Response-ProvideDDCCDocument.json');
+const storeHCRequest = require('../../features/fixtures/storeHealthCertificate/Bundle-DDCCDocument.json');
```

Comment:
This variable name is a bit confusing given the context in which it's used - the response from retrieving a health certificate that was previously stored.

 Also this is the only fixture that doesn't exist in the `test/` directory. Would you be able to use `test/fixtures/Bundle-DDCC-Document-example.json`? (I think the only difference is that that DDCC Document has a signature at the end). If you didn't want a signature in the response, it might be helpful to create an unsigned DDCC Document fixture in the `test/` directory.